setup.py

```diff
@@ -1,13 +1,40 @@
 from setuptools import setup
 
+LONG_DESCRIPTION = """
+# Hfinger - fingerprinting malware HTTP requests
+
+Tool for fingerprinting malware HTTP requests. Based on Tshark and written in Python3. Working prototype stage :-)
+
+It's main objective is to provide a representation of malware requests in a shorter form than printing whole request, 
+but still human interpretable. This representation should be unique between malware families, 
+what means that any fingerprint should be seen only for one particular family.
+
+Project's website: [https://github.com/CERT-Polska/hfinger](https://github.com/CERT-Polska/hfinger).
+"""
+
 setup(
     name="hfinger",
-    version="0.1.0",
-    description="Hfinger - fingerprinting HTTP requests stored in pcap files",
+    version="0.2.0",
+    description="Hfinger - fingerprinting malware HTTP requests stored in pcap files",
     author="Piotr Bialczak",
     author_email="piotrb@cert.pl",
-    packages=["hfinger", "hfinger.configs"],
-    package_data={"hfinger": ["configs/*.json", "configs/*.txt"]},
-    install_requires=["fnvhash", "python-magic"],
+    packages=["hfinger"],
+    license="GPLv3",
```

Comment:
license kwarg is meant to be used when none of the classifiers is appropriate (thanks, @psrok1 )

Reply:
Deleted in https://github.com/CERT-Polska/hfinger/commit/e5b2758b5a67f3ff619a9de2fa72e6408b5d35f8